cmd/main.go

```diff
@@ -24,30 +26,30 @@ func main() {
 
 	logger, err := zap.NewProduction()
 	if err != nil {
-		log.Fatalf("can't initialize zap logger: %v", err)
+		log.Fatalf("Can't initialize zap logger: %v", err)
 	}
 	defer func() {
 		if err := logger.Sync(); err != nil {
 			log.Fatalf("Error syncing logger: %v", err)
 		}
 	}()
 
-	r := gin.Default()
-	r.GET("/ping", func(c *gin.Context) {
+	engine := gin.Default()
+	engine.GET("/ping", func(c *gin.Context) {
 		c.JSON(http.StatusOK, gin.H{
 			"message": "pong",
 		})
 	})
 
-	nsGroup := r.Group("/namespaces")
+	group := engine.Group("/namespaces")
 	{
-		nsGroup.GET("/", routes.ListNamespaces(client, logger))
-		nsGroup.GET("/:name", routes.GetNamespace(client, logger))
-		nsGroup.POST("/", routes.CreateNamespace(client, logger))
-		nsGroup.DELETE("/:name", routes.DeleteNamespace(client, logger))
+		group.GET("/", routes.ListNamespaces(client, logger))
+		group.GET("/:name", routes.GetNamespace(client, logger))
+		group.POST("/", routes.CreateNamespace(client, logger))
+		group.DELETE("/:name", routes.DeleteNamespace(client, logger))
```

Comment:
I think the `nsGroup` naming is on purpose here since there's desire to break up the different groups based on different API routes (`namespaces`, `secrets`, etc...)

Reply:
Sure, thanks!